protocols/S1S2.txt

```diff
@@ -54,6 +54,9 @@ units {
 }
 
 model interface {
+    independent var units ms
```

Comment:
As noted in the comments, I'd rather not have to change protocols to get them working.

Reply:
Changes undone!